contracts/stake/src/contract.rs

```diff
@@ -152,4 +191,32 @@ impl StakeContract {
 
         bytes
     }
+
+    pub fn add_owner(&mut self, owner: PublicKey) -> Result<(), Error> {
+        if self.is_owner(&owner)? {
+            return Ok(());
+        }
+        self.owners.insert(owner, true)?;
+        Ok(())
+    }
+
+    pub fn is_owner(&self, owner: &PublicKey) -> Result<bool, Error> {
+        Ok(self.owners.get(owner)?.is_some())
+    }
+
+    pub fn check_ownership(&self, owner: &PublicKey) -> Result<bool, Error> {
```

Comment:
Shouldn't this be named `is_whitelisted`?

Reply:
Right, it's a leftover